src/platform_impl/linux/x11/mod.rs

```diff
@@ -658,6 +658,18 @@ impl<T: 'static> EventLoop<T> {
     }
 }
 
+impl<T> AsFd for EventLoop<T> {
+    fn as_fd(&self) -> BorrowedFd<'_> {
+        self.event_loop.as_fd()
```

Comment:
This will fail to compile on systems without `epoll` and `kqueue`, like AIX. I'm not sure if we support any of these systems but it should be written down somewhere.

Reply:
calloop doesn't support them, so no need to write down.